deepchecks/utils/builtin_datasets_utils.py

```diff
@@ -34,10 +38,14 @@ def read_and_save_data(assets_dir, file_name, url_to_file, file_type='csv', to_n
         elif file_type == 'npy':
             data = np.load(BytesIO(requests.get(url_to_file).content))
             np.save(assets_dir / file_name, data)
+        elif file_type == 'json':
+            data = json.loads(requests.get(url_to_file).content)
+            with open(assets_dir / file_name, 'w', encoding='utf-8') as f:
+                json.dump(data, f)
         else:
             raise ValueError('file_type must be either "csv" or "npy"')
 
-    if to_numpy:
+    if to_numpy and (file_type in {'csv', 'npy'}):
         if isinstance(data, pd.DataFrame):
             data = data.to_numpy()
         elif not isinstance(data, np.ndarray):
```

Comment:
what if to_numpy and json? Nothing happens? A bit weird. Either should be documented or resolved